src/libasr/utils2.cpp

```diff
@@ -25,6 +25,7 @@ bool read_file(const std::string &filename, std::string &text)
     ifs.read(&bytes[0], filesize);
 
     text = std::string(&bytes[0], filesize);
+    text.erase( std::remove(text.begin(), text.end(), '\r'), text.end() );
```

Comment:
This fixes the failure in the windows. I don't know whether this is the correct fix.

Reply:
The good news is that, this almost fixed the `run_tests` issue in the Windows.
Ie., Now, we can do `python run_tests.py`.
The only left out part to fix is to stop the printing `Creating library *.lib and object *.exp` in the runtime

Reply:
We can just filter it from the output on Windows. Do you want to submit a PR?

Reply:
Ok, this will slow down every file read.

Ideally we handle this in the tokenizer, which we have to extend to handle `\r` properly.

Why does this change have to be here? What in this PR requires this change?

Reply:
https://github.com/lcompilers/lpython/pull/1297#discussion_r1025918851